cartographer/mapping/internal/2d/ray_to_pixel_mask_test.cc

```diff
@@ -116,7 +116,7 @@ TEST(RayCastingTest, SteepLine) {
   EXPECT_TRUE(ray[3].isApprox(Eigen::Array2i({2, 4})));
 }
 
-TEST(RayCastingTest, FlatLine) {
+TEST(RayToPixelMaskTest, FlatLine) {
   Eigen::Array2i begin = {1, 1};
```

Comment:
maybe swap begin and end in one test to cover the other code path?

Reply:
The other code path is already tested in multiple other tests. So I don't think it is necessary here again.